google-cloud-video_intelligence/.yardopts

```diff
@@ -0,0 +1,8 @@
+--no-private
+--title=Google Cloud Video Intelligence API
+--exclude lib/google/cloud/video_intelligence/v1beta1
```

Comment:
Doesn't excluding this directory hide the GAPIC docs from the YARD output? Are we sure we want this? @quartzmo, can you confirm the docs are still correct?